storage/connectors/bigquery/src/main/java/feast/storage/connectors/bigquery/writer/BigQueryFeatureSink.java

```diff
@@ -189,6 +195,18 @@ private TableDefinition createBigQueryTableDefinition(FeatureSetProto.FeatureSet
         TimePartitioning.newBuilder(TimePartitioning.Type.DAY).setField("event_timestamp").build();
     log.info("Table partitioning: " + timePartitioning.toString());
 
+    FieldList fieldsList = FieldList.of(Collections.emptyList());
```

Comment:
I think `fieldList` variable here is somehow unused i.e. not used when creating the latest BigQuery `TableDefinition`.

This I believe will result in the existing table schema (if exists) to be ignored?

Update of `FeatureSet` with BigQuery store will fail in the following case?
> Error: Field 'col1' is missing in new schema

Existing schema in BigQuery:
| column name | type |
| ----- | -------|
| name | STRING |
| col1 | INT64 |
| col2 | FLOAT |

New `FeatureSet` to apply:
```yaml
kind: feature_set
spec:
  project: myproject
  maxAge: 3600s
  name: myfeatureset
  entities:
  - name: name
    valueType: STRING
  features:
  - name: col2
    valueType: FLOAT
  - name: col3
    valueType: INT64
```

Reply:
Unless this is the intended behaviour. Although `createBigQueryTableDefinition` method has this in the comment:
> ... If a table already exists, existing fields will be retained ...

Reply:
Yep, found this oversight today while implementing the e2e test and fixed it :) nice catch